shared/style/dialer/dialer.css

```diff
@@ -1,5 +1,12 @@
+* {
+  /* By default do not allow any user text selection, only enable for certain
+  parts on a per-element/region area */
+  -moz-user-select: none;
```

Comment:
We shouldn't need to do this because this should fall through if applied to the ```body``` or ```html``` elements.